docs/development/getting-started.md

```diff
@@ -1,32 +1,142 @@
-dependencies, or somewhere it will not be moved or accidentally deleted.
+## Using a dependency management system
 
-1. To begin, create a new Java Project in Eclipse, or click the name of the project
+### Maven
+
+This is the recommended way. Maven is not hard to learn and it makes dependency management really easy.
+
+Before you begin writing a plugin, you will need a ```pom.xml``` for the plugin. It is pertinent that this file be included.
+
+Example pom.xml file for maven:
+
+```xml
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <!-- Here is where you will substitute your plugin's information -->
+    <groupId>io.gomint.demo.testplugin</groupId>
+    <artifactId>myfirstplugin</artifactId>
+    <version>1.0-SNAPSHOT</version>
+
+    <!-- Here is where you will substitute your plugin's name and description -->
+    <name>My first Plugin</name>
+    <description>A demonstration and tutorial plugin for gomint's api</description>
+
+    <repositories>
+        <!-- This is the maven repository where gomint is published to  -->
+        <repository>
+            <id>ossrh</id>
+            <url>https://oss.sonatype.org/content/repositories/snapshots</url>
+        </repository>
+    </repositories>
+
+    <dependencies>
+        <!-- This is the GoMint API Maven Library which is necessary to create a plugin -->
+        <dependency>
+            <groupId>io.gomint</groupId>
+            <artifactId>gomint-api</artifactId>
+            <version>1.0.0-SNAPSHOT</version>
+            <scope>provided</scope>
+        </dependency>
+    </dependencies>
+</project>
+```
+
+### Gradle
+
+Here is the repository snippet:
+
+<Tabs
+  defaultValue="groovy"
+  values={[
+    { label: 'Groovy', value: 'groovy', },
+    { label: 'Kotlin', value: 'kotlin', },
+  ]
+}>
+<TabItem value="groovy">
+
+```groovy
+    maven { url "https://oss.sonatype.org/content/repositories/snapshots" }
+```
+
+</TabItem>
+<TabItem value="kotlin">
+    
+```kotlin
+    maven { url = uri("https://oss.sonatype.org/content/repositories/snapshots") }
+```
+
+</TabItem>
+</Tabs>
+
+and here is the dependencies snippet:
+
+<Tabs
+  defaultValue="groovy"
+  values={[
+    { label: 'Groovy', value: 'groovy', },
+    { label: 'Kotlin', value: 'kotlin', },
+  ]
+}>
+<TabItem value="groovy">
+
+```groovy
+    compile "io.gomint:gomint-api:1.0.0-SNAPSHOT"
+```
+
+</TabItem>
+<TabItem value="kotlin">
+    
+```kotlin
+    compile("io.gomint:gomint-api:1.0.0-SNAPSHOT")
+```
+
+</TabItem>
+</Tabs>
+
+## Downloading the API JARs manually
+
+You can download the latest API jar here: https://ci.janmm14.de/job/public%7Emcbe%7Egomint/lastSuccessfulBuild/artifact/gomint-api/target/
```

Comment:
We have a S3 bucket which holds the latest JARs: https://gomint-artifacts.s3.amazonaws.com/latest.zip

Reply:
I'm linking to API-only jars.
Maybe I should link to oss snapshot repo instead?